mobile-save-for-later/script/ci

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+sbt "project mobile-save-for-later; clean; compile; riffRaffUpload"
```

Comment:
This is currently running [via this build step](https://teamcity.gutools.co.uk/admin/editRunType.html?id=buildType:Mobile_MobileSaveForLater&runnerId=RUNNER_135&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253AMobile_MobileSaveForLater%26init%3D1&cameFromTitle=), which I've disabled for now.